### PR TITLE
Finished Implementing Admin and Student side of Task List.

### PR DESCRIPTION
**Pull request by navaneeth-arun**
-Made a post request from the click of a button (frontend) to store data in the backend in MongoDB.
-Made a get request on load of the page to display the list of tasks entered by the admin on page.
-Both changes were made with the help of @pankhuri813.
-Deleted unnecessary backend files that were used earlier for the practice of CRUD operations.
-Created a new schema that was deemed necessary for my current project.
-Updated the port for backend since it was causing unwanted errors.
-Made a delete request from the click of a button (frontend) to delete data from the backend in MongoDB.

### CODE REVIEW

File: Backend/.env [hunk 1]
```diff
@@ -1,2 +1,2 @@
-PORT=6969
+PORT=5000
```
Comment (PatelNeelMahesh) [inline Backend/.env]: Add `.env` in `.gitognore`, it should not be available on github.

**Review by PatelNeelMahesh — COMMENTED** [review]
You can merge pull request after making the required changes.

File: Backend/.env (hunk 1 above)
Comment (navaneeth-arun) [reply Backend/.env]: Done.

**Review by PatelNeelMahesh — APPROVED** [review]
You can merge pull request.

**Pull request merged**